### DISCUSSION TITLE
Is Android long-term supported platform?

### DISCUSSION THREAD

**Discussion by buger404** [ppy/osu#16207] [Q&A] [answered]
the user experience seems that it's far not complete on Android, something basic works poorly.

The default ui scale is too small.
It can run in portrait screen(I think it's a very inproper scale)
When I touched a object where I can text, the virtual keyboard appeared but it's not responsable to any keys.

Or we're not focusing on Android at this stage? Or it's designed for Android x86?

**Comment by bdach** [comment, marked as answer]
primary current focus is desktop platforms, and all of the issues you mention are known. will get better over time, but for now you can treat the android version as an early preview.
